app/assets/stylesheets/common/components/user-status-message.scss

```diff
@@ -1,7 +1,4 @@
 [data-content][data-identifier="user-status-message-tooltip"] {
-  display: flex;
-  align-items: center;
```

Comment:
first one is overridden by `.fk-d-tooltip`'s `display: flex !important`
the second one doesn't do anything as this element has effectively only one child (`.fk-d-tooltip__inner-content`), the other one is positioned absolutely.